src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting/HtmlFormattingTest.cs

```diff
@@ -387,6 +388,35 @@ public async Task FormatsComponentTags()
 tagHelpers: tagHelpers);
         }
 
+        [Fact]
+        [WorkItem("https://github.com/dotnet/aspnetcore/issues/26836")]
```

Comment:
Interesting, so your typical workflow associates every test with the "motivator" behind it? I'd imagine this would only be used when working with flaky / skipped tests.

In Roslyn have you found yourself back-referencing the work items in some fashion or removing them when tests are repurposed/modified?

Reply:
> so your typical workflow associates every test with the "motivator" behind it?

When a test is written because of an issue report, yes. :)

> In Roslyn have you found yourself back-referencing the work items in some fashion or removing them when tests are repurposed/modified?

Yes, before removing a test with this attribute I verify that the original scenario is either no longer relevant or is covered by something else.